src/service/alerts/mod.rs

```diff
@@ -207,17 +208,57 @@ pub async fn get_alert(
     }
 }
 
+#[tracing::instrument]
+pub async fn trigger_alert(
+    org_id: String,
+    stream_name: String,
+    stream_type: StreamType,
+    name: String,
+) -> Result<HttpResponse, Error> {
+    let result = db::alerts::get(
+        org_id.as_str(),
+        stream_name.as_str(),
+        stream_type,
+        name.as_str(),
+    )
+    .await;
+
+    match result {
+        Ok(Some(alert)) => {
+            let trigger = Trigger {
+                timestamp: Utc::now().timestamp_micros(),
+                is_valid: false,
+                alert_name: name,
+                stream: stream_name,
+                org: org_id,
+                last_sent_at: 0,
+                count: 0,
+                is_ingest_time: false, // Check with folks if this is correct
```

Comment:
is_ingest_time should be based on alert.is_real_time , when alert is created it is either real time or scheduled